clion-resharper/src/test/java/org/sonarlint/intellij/clion/resharper/CLionResharperAnalyzerConfigurationTest.java

```diff
@@ -55,6 +57,9 @@ void map_to_cfamily_compiler() {
     assertEquals("clang-cl", CLionResharperAnalyzerConfiguration.mapToCFamilyCompiler(ClangClCompilerKind.INSTANCE));
     assertEquals("msvc-cl", CLionResharperAnalyzerConfiguration.mapToCFamilyCompiler(MSVCCompilerKind.INSTANCE));
     assertEquals("clang", CLionResharperAnalyzerConfiguration.mapToCFamilyCompiler(AppleClangCompilerKind.INSTANCE));
+    OCCompilerKind iarCompilerKind = mock(OCCompilerKind.class);
+    when(iarCompilerKind.getDisplayName()).thenReturn("IAR");
+    assertEquals("iar", CLionResharperAnalyzerConfiguration.mapToCFamilyCompiler(iarCompilerKind));
```

Comment:
On resharper, I wasn't able to find the class `IarCompilerKind` in the jar, so I am using a mock for the test.